modal/functions.py

```diff
@@ -1219,12 +1220,14 @@ async def _call_function(self, args, kwargs) -> ReturnType:
             return  # type: ignore
 
     async def _call_function_nowait(self, args, kwargs) -> _Invocation:
+        # This feature flag allows users to put a large number of inputs
+        modal_spawn_extended = os.environ.get("MODAL_SPAWN_EXTENDED")
```

Comment:
Note that exposing this through our [config system](https://github.com/modal-labs/modal-client/blob/7c35e09d8150cb1ff504bc899340d84ebd4e5d10/modal/config.py#L199-L224) would have the benefit of centralizing all of the configurable / feature-flag behaviors so that they're easier to keep track of. The downside is that we don't currently have a clear distinction between config values that we expect users to set and config values that we use internally. But we do have a handful of feature flags in there currently. (Sorry should have mentioned this when originally bringing up environment variables).

Reply:
Ah, thanks for mentioning that - I haven't seen the config setup before. Yea that sounds good, I'll make those changes.

Reply:
ok, just pushed another commit using the config system. Much nicer using that!